src/data/mainnet/base-tokens-info.json

```diff
@@ -20,5 +20,158 @@
+    "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/CTSI.png",
+    "isNative": false,
+    "infoUrl": "https://www.coingecko.com/en/coins/cartesi"
+  },
+  {
+    "name": "Curve.Fi USD Stablecoin",
+    "symbol": "crvUSD",
+    "decimals": 18,
+    "address": "0x417ac0e078398c154edfadd9ef675d30be60af93",
+    "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/crvUSD.png",
+    "isNative": false,
+    "infoUrl": "https://www.coingecko.com/en/coins/crvusd"
+  },
+  {
+    "name": "SmarDex Token",
+    "symbol": "SDEX",
+    "decimals": 18,
+    "address": "0xfd4330b0312fdeec6d4225075b82e00493ff2e3f",
+    "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/SDEX.png",
+    "isNative": false,
+    "infoUrl": "https://www.coingecko.com/en/coins/smardex"
+  },
+  {
+    "name": "WEXO",
+    "symbol": "WEXO",
+    "decimals": 18,
+    "address": "0xac12f930318be4f9d37f602cbf89cd33e99aa9d4",
+    "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/WEXO.png",
+    "isNative": false,
+    "infoUrl": "https://www.coingecko.com/en/coins/wexo"
+  },
+  {
+    "name": "Toshi",
+    "symbol": "TOSHI",
+    "decimals": 18,
+    "address": "0xac1bd2486aaf3b5c0fc3fd868558b082a531b2b4",
+    "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/TOSHI.png",
+    "isNative": false,
+    "infoUrl": "https://www.coingecko.com/en/coins/toshi"
+  },
+  {
+    "name": "Wuffi",
+    "symbol": "WUF",
+    "decimals": 4,
+    "address": "0x4da78059d97f155e18b37765e2e042270f4e0fc4",
+    "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/WUF.png",
+    "isNative": false,
+    "infoUrl": "https://www.coingecko.com/en/coins/wuffi"
+  },
+  {
+    "name": "LimeWire Token",
+    "symbol": "LMWR",
+    "decimals": 18,
+    "address": "0xe997017e0cb0ceb503565f181e9ea922cd979c35",
+    "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/LMWR.png",
+    "isNative": false,
+    "infoUrl": "https://www.coingecko.com/en/coins/limewire-token"
+  },
+  {
+    "name": "Dola USD Stablecoin",
+    "symbol": "DOLA",
+    "decimals": 18,
+    "address": "0x4621b7a9c75199271f773ebd9a499dbd165c3191",
+    "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/DOLA.png",
+    "isNative": false,
+    "infoUrl": "https://www.coingecko.com/en/coins/dola-usd"
+  },
+  {
+    "name": "Degen",
+    "symbol": "DEGEN",
+    "decimals": 18,
+    "address": "0x4ed4e862860bed51a9570b96d89af5e1b0efefed",
+    "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/DEGEN.png",
+    "isNative": false,
+    "infoUrl": "https://www.coingecko.com/en/coins/degen-base"
+  },
+  {
+    "name": "Propy",
+    "symbol": "PRO",
+    "decimals": 18,
```

Comment:
Think this token is 8 decimals.

Reply:
Hmm, you're right. Now i feel the need to double check all of them